keps/sig-cluster-lifecycle/draft-20180316-kubeadm-phases-beta.md

```diff
@@ -0,0 +1,243 @@
+# kubeadm phases to beta (GA)
+
+## Metadata
+
+```yaml
+---
+kep-number: draft-20180316
+title: kubeadm phases to beta (GA)
+status: accepted
+authors:
+  - "@fabriziopandini"
+owning-sig: sig-cluster-lifecycle
+reviewers:
+  - TBD
+approvers:
+  - TBD
+editor:
+  - "@fabriziopandini"
+creation-date: 2018-03-16
+last-updated: 2018-03-16
+see-also:
+
+```
+
+## Table of Contents
+
+  * [kubeadm phases to beta (GA)](#kubeadm-phases-to-beta-ga)
+    * [Metadata](#metadata)
+    * [Table of Contents](#table-of-contents)
+    * [Summary](#summary)
+    * [Motivation](#motivation)
+        * [Goals](#goals)
+        * [Non-goals](#non-goals)
+        * [Challenges and Open Questions](#challenges-and-open-questions)
+    * [Proposal](#proposal)
+        * [User Stories](#user-stories)
+          * [Invoke a specific task/phase of the kubeadm init workflow](#invoke-a-specific-taskphase-of-the-kubeadm-init-workflow)
+        * [Implementation Details](#implementation-details)
+          * [Expected benefits](#expected-benefits)
+          * [Phased Command Builder](#phased-command-builder)
+          * [cmdUtil factory](#cmdutil-factory)
+    * [Graduation Criteria](#graduation-criteria)
+    * [Implementation History](#implementation-history)
+    * [Drawbacks](#drawbacks)
+    * [Alternatives](#alternatives)
+
+## Summary
+
+We are defining the roadmap for graduating `kubeadm alpha phase` commands to
+beta (and GA), addressing concerns/lessons learned so far about the additional
+effort for maintenance of this feature.
+
+## Motivation
+
+The `kubeadm phase` command was introduced in v1.8 cycle under `kubeadm alpha phase`
+command with the goal to provide to users an interface for invoking individually any
+task/phase of the `kubeadm init` workflow.
+
+During this period,  `kubeadm phase`  proved to be a valuable and composable
+API/toolbox that can be used by any IT automation tool or by advanced user for
+creating custom clusters.
+
+On the other side, it was necessary a continuous effort for maintaining this
```

Comment:
this sentence is a little confusing. "It required a continuous effort to keep this component up to date with" maybe?